css/index.css

```diff
@@ -51,6 +51,83 @@ table {
     box-sizing: border-box;
 }
 
+
+.image1 {
+  max-width:100%;
+  max-height:100%;
+}
+
+.topsec {
+  display: flex;
+  justify-content: space-between;
+  height: 50px;
+}
+
+.topimg {
+  width: 20%;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.topsec nav {
+  display: flex;
+  width: 80%;
+  justify-content: space-around;
+  align-items: flex-end;
+}
+
+.topsec nav a {
+  text-decoration: none;
+  color: black;
+}
+
+.image2 {
+  width: 100%;
+  height: 190px;
+  padding-top: 30px;
+  padding-bottom: 20px;
+}
+
```

Comment:
Excellent job only changing the color in the child and letting the parent do the rest of the styling.